.github/workflows/build-and-deploy.yaml

```diff
@@ -0,0 +1,76 @@
+on:
+  push:
+    branches:
+      - main
+
+name: Deploy to AWS Production (K8s)
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: production
+    env:
```

Comment:
I suggest putting SERVICE_NAME and maybe ENVIRONMENT_TAG up here as well. So it's not needed to edit this file all that much between services and environments, therefore less chances for typos

Reply:
@eskp done